deploy/internal/nsfs-pvc-cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: default
+spec:
+  storageClassName: default
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
also here, write the size that we intend to use - 30GB. and add a new line in the end.

Reply:
done.